spec/rails_helper.rb

```diff
@@ -75,4 +75,16 @@
     with.test_framework :rspec
     with.library :rails
   end
+
+  class RequestObj
```

Comment:
I tried that, but it didn't work well as some methods didn't work correctly. An example is the `secrete`  method.